src/tiles/tile-group.component.ts

```diff
@@ -0,0 +1,95 @@
+import {
+	Component,
+	AfterContentInit,
+	Input,
+	Output,
+	EventEmitter,
+	HostBinding,
+	ContentChildren,
+	QueryList
+} from "@angular/core";
+import { SelectionTile } from "./selection-tile.component";
+import { NG_VALUE_ACCESSOR } from "@angular/forms";
+
+@Component({
+	selector: "ibm-tile-group",
+	template: `<ng-content select="ibm-selection-tile"></ng-content>`,
+	providers: [
+		{
+			provide: NG_VALUE_ACCESSOR,
+			useExisting: TileGroup,
+			multi: true
+		}
+	]
+})
+export class TileGroup implements AfterContentInit {
+	static tileGroupCount = 0;
+	/**
+	 * The tile group `name`
+	 */
+	@Input() name = `tile-group-${TileGroup.tileGroupCount}`;
+	/**
+	 * Set to `"multi"` to support multiple tile selection
+	 */
+	@Input() type: "single" | "multi" = "single";
```

Comment:
I'd like to start moving everything to `multiple` instead of `type: "single" | "multi"` for multiple selections ... primarily to match existing html semantics